CHANGELOG.md

```diff
@@ -8,9 +8,40 @@ For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/
   Full Keycloak upstream jira issue can be shown if filtered by Fix version. For example [Keycloak jira issue for 15.0.2 version](https://issues.redhat.com/browse/KEYCLOAK-19161?jql=project%20%3D%20keycloak%20and%20fixVersion%20%3D%2015.0.2)
```

Comment:
Replace introduction with:
```
# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Reply:
This exists. So remove lines 7-8?

Reply:
You're right - then let's remove lines 7-8